scripts/standard_configure.bat

```diff
@@ -39,13 +36,20 @@ del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"
 rem Configure
 @echo on
 cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"          ^
--DUSE_EIGEN_MKL=OFF
+-DINCLUDE_FEAST=OFF                                                                                 ^
```

Comment:
please do not commit the changes to the core
even if you revert them, they will show up in the history and this makes debugging and tracing errors in the history more difficult

To solve this please do the following:
1. revert the changes
2. Finish your changes
3. [Squash](https://github.blog/2016-04-01-squash-your-commits/) this PR instead of a regular merge. This is the only way that the changes will not show up in the history

Reply:
I'm sorry that this is a problem. Normally i do random commits for myself and at the end i do a complete rebase and don't commit the useless intermediate stuff. Then the changes there should also vanish from the history, right?

Reply:
hm ok
I am not 100% sure that the rebase will solve the problem. Doesn't it apply the (same) commits to the master (or whatever branch you do the rebase with)?

Reply:
My fault. I forgot to mention half of my usual step. First i rebase onto master. Then i reset my branch to the master commit but keep local changes. Then i write new commits for local changes and neglect stupid ones like the one above.
After this i do a force-push to origin. This is the procedure i wanted to discuss. Sorry for the inconvenience!